am-lib/src/main/java/uk/gov/hmcts/reform/amlib/enums/Permission.java

```diff
@@ -42,11 +45,31 @@ public static int sumOf(Set<Permissions> perms) {
 
     /**
      * Performs a binary AND operation to determine weather the 'permissions' value has suitable permissionToCheck.
-     * @param permissions the decimal value of permissions defined in Permissions enum
+     *
+     * @param permissions       the decimal value of permissions defined in Permission enum
      * @param permissionToCheck the permission to verify
      * @return Returns true if the binary AND of the provided 'permissions' and 'permissionToCheck' is true.
      */
-    public static boolean hasPermissionTo(int permissions, Permissions permissionToCheck) {
+    public static boolean hasPermissionTo(int permissions, Permission permissionToCheck) {
         return (permissions & permissionToCheck.getValue()) == permissionToCheck.getValue();
     }
+
+    /**
+     * Builds a list of permissions based on decimal value.
```

Comment:
IMO it is not decimal as there is no decimal fraction. Shall we say numeric or integer?